### PR TITLE
Adding concepts for Dual Duct HVAC systems

### PR DESCRIPTION
**Pull request by lazlop**
Added concepts for dual duct HVAC systems as mentioned in #376

### CODE REVIEW

**Comment by gtfierro** [comment]
@lazlop let me know when you get a chance to come back to this and make the edits we discussed on the ontology working group call last week!

**Comment by lazlop** [comment]
@gtfierro Sorry, the end of March has been pretty busy and I just got around to this! This is my first attempt at writing a SHACL shape, let me know if it looks right. Also, let me know where the location of the SHACL shape should be. Just added it to BrickEntityShapeBase.ttl for now

File: bricksrc/equipment.py [hunk 1]
```diff
@@ -640,6 +642,14 @@
                 "tags": [TAG.Equipment, TAG.RTU],
                 OWL.equivalentClass: BRICK["Rooftop_Unit"],
             },
+            "Dual_Duct_Air Handling Unit": {
```
Comment (gtfierro) [inline bricksrc/equipment.py:645]: You will need underscores instead of spaces here

File: shacl/BrickEntityShapeBase.ttl [hunk 2]
```diff
@@ -160,3 +160,25 @@ bsh:hasLocationShape a sh:Nodeshape;
     ];
     sh:message "Points are a virtual concept and always belonging to a physical device, represented by Equipment. Thus, it cannot have a Location alone."
     .
+
+bch:hasHotColdDeck
```
Comment (gtfierro) [inline shacl/BrickEntityShapeBase.ttl:164]: This should be 'bsh' instead of 'bch'

**Review by gtfierro — CHANGES_REQUESTED** [review]
No worries @lazlop! I made a couple small suggestions but I think this is very close

**Review by gtfierro — APPROVED** [review]

**Comment by gtfierro** [comment]
Thanks @lazlop!

**Pull request merged**